LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -349,8 +352,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </item>
                                 <item xml:id="e3">
                                     <locus target="#2r"/>
-                                    <desc type="findingAid">Note by a later Ethiopian hand indicating the main text: <foreign xml:lang="gez">ኢሳይያስ፡
-                                        ዘቅዱስ፡ መድኃኒ፡ ዓለም።</foreign></desc>
+                                    <desc type="OwnershipNote">Note by a later Ethiopian hand: <foreign xml:lang="gez">ኢሳይያስ፡ ዘቅዱስ፡ 
```

Comment:
Yes, this was my first estimation. It is supposed to be the "(Book of) Isaiah from (the church) of Madḫāne ʿAlam. I will encode the book as a book (with its LIT) to avoid future misinterpretation and the church as a repository (with INS0101MadhaneAlam). This manuscript was one of those, that have been looted from Maqdala. For this reason, it is very likely, that INS0101MadhaneAlam is meant. I think, the tag "OwnershipNote" is still justified as it refers to the property of the church.